docs/dev/topology-over-raft.md

```diff
@@ -22,15 +22,81 @@ Nodes in state `left` may still appear as tablet replicas in host_id-based repli
 State transition diagram for nodes:
 ```mermaid
 stateDiagram-v2
-    none --> bootstrapping|replacing
+    none --> bootstrapping: join
+    none --> replacing: replace
+    state bootstrapping {
+        bs_join_group0 : join_group0
+        bs_left_token_ring : left_token_ring
+        bs_commit_cdc_generation : commit_cdc_generation
+        bs_write_both_read_old : write_both_read_old
+        bs_write_both_read_new : write_both_read_new
+        [*] --> bs_join_group0
+        bs_join_group0 --> bs_left_token_ring: rollback
```

Comment:
rollback can be entered from commit_cdc_generation and write_both_read_old as well.